internal/extract/pod_spec.go

```diff
@@ -18,6 +19,8 @@ func PodTemplateSpec(obj k8sutil.Object) (coreV1.PodTemplateSpec, bool) {
 		}, true
 	case *batchV1Beta1.CronJob:
 		return obj.Spec.JobTemplate.Spec.Template, true
+	case *appsV1.Deployment:
+		return obj.Spec.Template, true
```

Comment:
You don't need to do this, this is automatically handled using reflection in the `default` case below.

Reply:
Done